ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -58,6 +60,8 @@ object Config {
       seeding = Seeding.Strong,
       metricsReporter = None,
       metricsReportingInterval = Duration.ofSeconds(10),
+      noIndexer = false,
+      noLedgerApiServer = false,
```

Comment:
I assume that two `false`s here would make no sense, right? Would it be better to have an enumeration of cases and choose from those?

Probably you can extend `Mode` to have `RunFullParticipant`, `RunIndexerOnly` and `RunApiServerOnly` and have a CLI option that reflects that, like `--mode=[full,indexer-only,api-server-only]`, with `full` being the default.